test/test_data_dmri.py

```diff
@@ -182,6 +182,46 @@ def test_equality_operator(tmp_path):
     assert round_trip_dwi_obj == dwi_obj
 
 
+def test_shells(datadir):
+    dwi_h5 = load(datadir / "dwi.h5")
+    num_bins = 3
```

Comment:
Can we use the bvecs/bvals files we added for testing? They are within the repo (i.e., no need to pull extra data from GIN) and they provide several use-cases ideal for testing this.

Reply:
Not sure if I understand: `shells` is an [instance method](https://github.com/nipreps/nifreeze/pull/74/files#diff-774e30bff24e702cda5516cbc6bbbcae1af21b710e2b0ad22cc15741fbed9afeR245) and thus we need a `DWI` instance. Are you proposing to build a `DWI` instance using random data? I'd say that this would add unnecessary complexity to the test, unless that instance can be generalized and be built to be used by other tests. TBH, I would open an issue if my interpretation is correct and leave that for a separate PR.

Reply:
390a139 creates random data for the DWI dataobj and uses the bval/bvec data from the repository.

Reply:
I am resolving this. I am cross-referencing https://github.com/nipreps/nifreeze/pull/71#issuecomment-2762934819. Once I rebase PR #71 on `main` I'll try to figure out how to merge/refactor the random testing data creation methods to a common place.